services/README.md

```diff
@@ -0,0 +1,5 @@
+# Services
```

Comment:
why did you not touch the API-service/README, unlike the IBF-dashboard/README? In general, did you go over all remaining README's in the repo and check them?
- I see one in tools/db-dumps/ which I cannot imagine is still used. Remove it?
- The rest seems fine actually.

Reply:
I couldn't get to it within this task. It should be the next documentation PBI before the folder reorg mentioned above. I definitely missed some READMEs.

I removed db-dumps https://github.com/rodekruis/IBF-system/pull/2120/commits/03f6689694aa1ea2ffcbf4137251f80b705e6d17